Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
inlines? Well, methods inside class declarations are inline by default. That's why I wrote almost all methods for MBL within the class (because they also were pretty short). So really adding inlines there is over-specifying it, and I consider that to be unnecessary clutter. The FORCE_INLINE is both an "inline" specifier + a gcc compile attribute, which when I read about it seem to be more aimed at functions. And what we can see in the behavior that the compiler actually does a pretty good job of select size over inline when necessary. I imagine different optimize options will tune the values in either direction.